application/console/migrations/m230515_162100_create_session_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use yii\db\Migration;
+
+class m230515_162100_create_session_table extends Migration
+{
+    // Use safeUp/safeDown to run migration code within a transaction
+    public function safeUp()
+    {
+        /*
+         * UserTable
+         */
+        $this->createTable(
+            '{{session}}',
+            [
+                'id' => 'char(40) primary key not null',
+                'expire' => 'int(11) null',
+                'data' => 'binary not null',
```

Comment:
```suggestion
                'data' => 'blob not null',
```
According to the [DbSession docs](https://www.yiiframework.com/doc/api/2.0/yii-web-dbsession#$sessionTable-detail) this should be a `LONGBLOB`. I'm not sure we ever store that much data in the session, but `binary` is much shorter (255 bytes).